src/lib/components/commons/Badge/Badge.tsx

```diff
@@ -1,14 +1,15 @@
 import { PropsWithChildren } from 'react';
-import { StyledBadge } from './Badge.styled';
+import { StyledBadge, StyledColumnBadge } from './Badge.styled';
 import { IBadgeProps } from './Badge.types';
 
 export const Badge = ({
   children,
   ...props
```

Comment:
if the only prop is `columnsBadge` why not destructor  it at the top level... why make this destructor twice?